packages/botonic-react/src/webchat/messages-reducer.js

```diff
@@ -14,7 +14,10 @@ export const messagesReducer = (state, action) => {
     case ADD_MESSAGE_COMPONENT:
       return {
         ...state,
-        messagesComponents: [...state.messagesComponents, action.payload],
+        messagesComponents: [
+          ...(state.messagesComponents || []),
+          action.payload,
+        ],
```

Comment:
Why the message ID duplicate check is not included in this PR?

Reply:
I tried to added it here, but the logic we added was not working correctly. I prefer to close this PR which is getting large and keep investigating this issue aside in another PR (I added a task for it).

Reply:
added 👍